src/baguetteBox.js

```diff
@@ -313,8 +327,19 @@
         if (options.buttons === 'auto' && ('ontouchstart' in window || imagesMap[currentGallery].length === 1)) {
             options.buttons = false;
         }
-        // Set buttons style to hide or display them
-        previousButton.style.display = nextButton.style.display = (options.buttons ? '' : 'none');
+
+        if (options.buttons === false) {
+            previousButtonContent.setAttribute('aria-hidden', 'true');
+            nextButtonContent.setAttribute('aria-hidden', 'true');
+
+            // Set buttons style to hide
+            previousButton.style.display = nextButton.style.display = 'none';
+        } else {
+            previousButtonContent.setAttribute('aria-hidden', 'false');
+            nextButtonContent.setAttribute('aria-hidden', 'false');
+            closeButtonContent.setAttribute('aria-hidden', 'false');
+        }
```

Comment:
as you have `aria-label` on the button, which should override anything inside the button anyway as far as accessible name exposed to AT is concerned, you could even save yourself the hassle and keep it as `x` instead of `&times;` (though visually the latter will look nicer ;) )